dotnet/src/SemanticKernel/Orchestration/SKFunction.cs

```diff
@@ -112,12 +112,11 @@ public CompleteRequestSettings RequestSettings
                 string completion = await client.CompleteAsync(prompt, requestSettings);
                 context.Variables.Update(completion);
             }
-#pragma warning disable CA1031 // We need to catch all exceptions to handle the execution state
-            catch (Exception e) when (!e.IsCriticalException())
+            catch (Exception ex) when (!ex.IsCriticalException())
             {
-                context.Fail(e.Message, e);
+                log?.LogError(ex, "Something went wrong loading semantic function from config: {0}.{1}. Error: {2}", skillName, functionName, ex.Message);
```

Comment:
the log msg is incorrect. The error could be about rendering, about execution, etc. The config is not involved. 

RenderAsync and CompleteAsync should already log errors internally, so this might be redundant

Reply:
@dluc, changed the message to be a warning using the summary description as a reference. 

I think is a good practice to logging when possible when an exception happened. 

Not logging in this context could hide a problem and turn the troubleshooting more difficult.